packages/turf-voronoi/index.js

```diff
@@ -0,0 +1,45 @@
+import lineStringToPolygon from '@turf/linestring-to-polygon';
+import { lineString } from '@turf/helpers';
+import d3voronoi from 'd3-voronoi';
+
+function und3ify(polygon) {
+    return lineStringToPolygon(lineString(polygon.slice(0)));
+}
+
+/**
+ * Takes a FeatureCollection of points, and a bounding box, and returns a FeatureCollection
+ * of Voronoi polygons. 
+ * 
+ * The Voronoi algorithim used comes from the d3-voronoi package.
+ *
+ * @name voronoi
+ * @param {FeatureCollection<Point>|Feature[]<Point>} points to find the Voronoi polygons around.
+ * @param {number[]} bbox clipping rectangle, in [minX, minY, maxX, MaxY] order.
+ * @returns {FeatureCollection<Polygon} a set of polygons, one per input polygon.
+ * @example
+ * var bbox = [-70, 40, -60, 60];
+ * var points = turf.random('points', 100, { bbox: bbox }); 
```

Comment:
The `random` module has changed in the newest release.

```
var points = turf.randomPoints(100, { bbox: bbox }); 
```

@stebogit I guess we could re-implement the `random()` method with the legacy behavior.